docs/streams/developer-guide/config-streams.html

```diff
@@ -70,9 +70,11 @@
           <li><a class="reference internal" href="#optional-configuration-parameters" id="id6">Optional configuration parameters</a>
             <ul>
               <li><a class="reference internal" href="#acceptable-recovery-lag" id="id27">acceptable.recovery.lag</a></li>
-              <li><a class="reference internal" href="#default-deserialization-exception-handler" id="id7">default.deserialization.exception.handler</a></li>
+              <li><a class="reference internal" href="#default-deserialization-exception-handler" id="id7">default.deserialization.exception.handler (deprecated)</a></li>
+              <li><a class="reference internal" href="#deserialization-exception-handler" id="id42">deserialization.exception.handler</a></li>
```

Comment:
We should insert in alphabetical order

Reply:
We should also insert new rows with a detailed description (guess we can just c&p?) -- otherwise, we generate dangling links here.